cpu/native/include/cpu_conf.h

```diff
@@ -55,6 +55,11 @@ extern "C" {
  */
 #define NATIVE_ETH_PROTO 0x1234
 
+#if GNRC_PKTBUF_SIZE < 2048
+#   undef  GNRC_PKTBUF_SIZE
+#   define GNRC_PKTBUF_SIZE     (2048)
```

Comment:
I thought about it, but a) we're dealing with native here, so additional memory doesn't hurt and b) we would still have to touch all the examples and c) not erroring is friendlier. ;-)